rllib/examples/policy/rock_paper_scissors_dummies.py

```diff
@@ -38,7 +40,7 @@ def compute_actions(
         episodes=None,
         **kwargs
     ):
-        return state_batches[0], state_batches, {}
+        return [state_batches[0][0] for x in obs_batch], state_batches, {}
```

Comment:
Just like BeatLastHeuristic, we need to provide one actions per obs here.
Env runner produces partially empty MA action dicts otherwise.